src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -1,19 +1,46 @@
 package core.basesyntax.impl;
 
 import core.basesyntax.Storage;
+import java.util.Objects;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+
+    private int size;
+    private final int max = 10;
```

Comment:
```suggestion
    private static final int MAX_SIZE = 10;
```
якщо це константа, то її треба капсом писати